front/app/components/admin/ContentBuilder/Widgets/ImageMultiloc/index.tsx

```diff
@@ -37,12 +37,14 @@ import useLocalize from 'hooks/useLocalize';
 import InputMultilocWithLocaleSwitcher from 'components/UI/InputMultilocWithLocaleSwitcher';
 
 interface Props {
-  imageUrl?: string;
+  image?: {
```

Comment:
We need to change how the image element works and make it consistent for the homepage banner in order to facilitate the back-end when using templates.